src/PrestaShopBundle/ApiPlatform/Resources/readme.md

```diff
@@ -0,0 +1,3 @@
+# Do not delete this folder it is needed for APIPlatform conf.
```

Comment:
maybe a `.gitkeep` is more appropriated?

Reply:
Jo told me to add a readme here.

Reply:
Is there any official recommandation from ApiPlatform?